features/step_definitions/run_steps.rb

```diff
@@ -14,12 +14,14 @@
 
 
 When(/^I run `(.+?)` and enter "(.+?)"$/) do |command, input|
-  @result = run command, input: input
+  inputs = Kappamaki.from_sentence(input)
+  @result = run command, inputs: inputs
```

Comment:
I'd say that words like "sand", "water", or "input" have no meaningful plural. They imply multiple individual things.